config/locales/controllers/en.yml

```diff
@@ -1,12 +1,21 @@
 ---
 en:
+  blocked:
+    users:
+      create:
+        blocked: "You have blocked the user %{name}."
+      destroy:
+        unblocked: "You have unblocked the user %{name}."
   chapters:
     anonymous: Anonymous
     chapter_position: " - Chapter %{position}"
     multifandom: Multifandom
     unrevealed: Mystery Work
     unspecified_fandom: No fandom specified
   comments:
+    check_blocked:
+      parent: "Sorry, you have been blocked by one or more of this work's owners."
+      reply: "Sorry, that commenter has blocked you."
```

Comment:
We'd like to go with `Sorry, you have been blocked by that user.` in the places that currently use this phrasing.